apps/ftu/test/unit/navigation_test.js

```diff
@@ -315,8 +315,50 @@ suite('navigation >', function() {
       });
       observer.observe(UIManager.mainTitle, observerConfig);
     });
+
   });
 
+  suite('Dogfood Settings>', function() {
+    var observerConfig = {
+      attributes: true
+    };
+
+    setup(function() {
+      // Needed to make sure that there is a DOM change to trigger
+      // the MutationObserver.
+      setStepState(1);
+    });
+
+    teardown(function() {
+    });
+
+    test('metrics checkbox should disabled for dogfooders > ', function(done) {
+      Navigation.isDogfooder = true;
```

Comment:
I think that instead of setting the `Navigation.isDogFooder` directly, we should modify the setting and check the whole process.